rpm/tracker-miners.spec

```diff
@@ -93,11 +95,14 @@ ln -s ../tracker-miner-fs-3.service %{buildroot}%{_userunitdir}/post-user-sessio
 cp -a %{SOURCE1} %{buildroot}%{_datadir}/tracker3-miners/extract-rules/
 cp -a %{SOURCE2} %{buildroot}%{_datadir}/tracker3-miners/extract-rules/
 
+install -D -m 755 %{SOURCE3} %{buildroot}/%{_oneshotdir}/tracker-reset.sh
+
 %find_lang tracker3-miners
 
 %post
 /sbin/ldconfig
 glib-compile-schemas   /usr/share/glib-2.0/schemas/
+add-oneshot --now --new-users --all-users tracker-reset.sh || :
```

Comment:
Why `--new-users`? What data they would have to migrate?

Reply:
Doesn't, but the script also creates the marker file. Otherwise the new users would get their databases reset on the next update for no reason.

Reply:
Ah, yeah, that's a very good point. :+1: